api/chaos/serverexperimentation/v1/serverexperimentation.proto

```diff
@@ -56,3 +56,128 @@ enum FaultInjectionCluster {
   // Faults are injected to ingesss traffic by an upstream service(s).
   FAULTINJECTIONCLUSTER_UPSTREAM = 2;
 }
+
+// The configuration of an HTTP fault.
+message HTTPFaultConfig {
+  oneof fault {
+    option (validate.required) = true;
+
+    // The abort fault.
+    AbortFault abort_fault = 1;
+    // The latency fault.
+    LatencyFault latency_fault = 2;
+  }
+}
+
+// The definition of an abort fault.
+message AbortFault {
+  // The targeting of the fault describing what requests faults should be applied to.
```

Comment:
I would try to phrase this more about "what requests are being considered for faults", so that it makes more sense with the fault_percentage field